src/_guides/language/analysis-options.md

```diff
@@ -402,4 +410,5 @@ Use the following resources to learn more about static analysis in Dart:
 [linter rules]: http://dart-lang.github.io/linter/lints/
 [sound-dart]: /guides/language/sound-dart
 [todo]: {{site.pub-api}}/analyzer/latest/analyzer/TodoCode/TODO-constant.html
-[use a specific version]: https://github.com/dart-lang/pedantic/blob/master/README.md#using-the-lints
+[specific version of pedantic,]: https://github.com/dart-lang/pedantic/blob/master/README.md#using-the-lints
+[pedantic analysis options file,]: https://github.com/dart-lang/pedantic/tree/master/lib
```

Comment:
I think the original link is better--at the end of the section we show how to depend on a specific version of the yaml file.

https://github.com/dart-lang/pedantic/blob/master/README.md#using-the-lints

They do still need to add the dependency. `^1.7.0` means 'use anything compatible with 1.7.0' so it gets around the version incompatibility problem I described.

Reply:
I realized that it's probably best to link to the pub.dev version of the README, instead of the github blob, so now I'm linking to https://pub.dev/packages/pedantic#using-the-lints.